fintrack/transactions/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+from choices import TransactionTypeChoices
+
+
+class Transaction(models.Model):
+
+    description = models.CharField(max_length=300, blank=True)
+    category = models.CharField(max_length=20, choices=TransactionTypeChoices.choices, blank=False)
```

Comment:
either use category or type not both, while writing code make sure to be consistent